persistence/sql/migrations/20241609000001000000_device_flow.postgres.down.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_challenge_id_fk;
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_client_id_fk;
+ALTER TABLE hydra_oauth2_device_auth_codes DROP CONSTRAINT IF EXISTS hydra_oauth2_device_auth_codes_nid_fk_idx;
+
+DROP TABLE IF EXISTS hydra_oauth2_device_auth_codes;
+
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_challenge_id;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_code_request_id;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_verifier;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_csrf;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_was_used;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_handled_at;
+ALTER TABLE hydra_oauth2_flow DROP COLUMN IF EXISTS device_error;
+
+ALTER TABLE hydra_client DROP COLUMN device_authorization_grant_id_token_lifespan;
+ALTER TABLE hydra_client DROP COLUMN device_authorization_grant_access_token_lifespan;
+ALTER TABLE hydra_client DROP COLUMN device_authorization_grant_refresh_token_lifespan;
```

Comment:
Why are these not `IF EXISTS`?

Reply:
I copied the file from the default version. I see no reason for that

Reply:
Updated both files to include it